test/support/api_case.ex

```diff
@@ -110,16 +110,17 @@ defmodule CodeCorps.ApiCase do
         conn |> post(path, payload)
       end
 
-      def request_update(conn, :not_found), do: conn |> request_update(-1, %{})
-      def request_update(conn, attrs \\ %{}), do: conn |> request_update(default_record, attrs)
+      def request_update(conn), do: request_update(conn, %{})
```

Comment:
@sbatson5 I believe this was what we had and were receiving a warning for it because of line 114.